cmd/cluster-operator-controller-manager/app/controllermanager.go

```diff
@@ -538,6 +539,19 @@ func startClusterController(ctx ControllerContext) (bool, error) {
 	return true, nil
 }
 
+func startClusterAPIClusterController(ctx ControllerContext) (bool, error) {
+	if !clusterOperatorResourcesAvailable(ctx) {
+		return false, nil
+	}
+	go cluster.NewClusterAPIController(
+		ctx.ClusterAPIInformerFactory.Cluster().V1alpha1().Clusters(),
+		ctx.ClusterAPIInformerFactory.Cluster().V1alpha1().MachineSets(),
+		ctx.ClientBuilder.KubeClientOrDie("clusterapi-cluster-controller"),
+		ctx.ClientBuilder.ClusterAPIClientOrDie("clusterapi-cluster-controller"),
+	).Run(int(ctx.Options.ConcurrentClusterAPIClusterSyncs), ctx.Stop)
```

Comment:
I've just been re-using the non-cluster-api values, as we're going to kill off the distinction shortly. Logical to just re-use cluster syncs here?

Reply:
sgtm